core/database_config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I don't think showing/pushing passwords in plain-text is a good idea 😅 

One idea is to add this config to the Configuration Manager's setup and make the user enter it manually. I already added a `.gitignore` entry for the `core/configuration.ini` so we won't push passwords like this by mistake in the future -- we can have our own `.ini` that we use locally but never commit to version control.

As for the default value, we can leave it as empty for now. We may have to look into using Environment Variables in the future and/or making setup questions that are **required** (i.e., user won't be able to press enter and dismiss the question and the Configuration Manager won't continue until that question is answered).

Also, make sure to change the password of the DB if this is the actual password 🙂

Reply:
Oops I totally forgot this repo was public -_-

Will address!

Reply:
So a followup problem would be that the tests I wrote for DB Client use the DB. As of now, I have separate credentials in the `tests/artifacts/db_client` directory. So I propose two potential solutions:

1. Remove tests for DB Client. If we can't publish the password to version control, then there isn't any way for the tests to access the DB, right?

2. Modify tests to mock out DB. This would mean just having the table pulled locally as a csv so that the "get data provider by category" functionality can be tested.

Ideally, 2 would be great. But I also think it's overkill since most of the DB Client's functionality is running SQL commands on the DB. 

What do you think?